classes/system/influxdb/server/single.yml

```diff
@@ -1,13 +1,32 @@
 classes:
 - service.influxdb.server.single
 parameters:
+  _param:
+    influxdb_home_directory: /var/lib/influxdb
+    influxdb_wal_directory: ${_param:influxdb_home_directory}/wal
   linux:
     system:
       repo:
         influxdb:
           enabled: true
           source: 'deb https://repos.influxdata.com/ubuntu xenial stable'
           key_url: 'https://repos.influxdata.com/influxdb.key'
+      user:
+        influxdb:
+          enabled: true
+          system: true
+          groups: influxdb
+          home: ${_param:influxdb_home_directory}
+    storage:
+      mount:
+        tmpfs:
+          enabled: true
+          device: tmpfs
+          file_system: tmpfs
+          path: ${_param:influxdb_wal_directory}
+          opts: size=4294967296,rw
```

Comment:
is it really 4GB? hardcoded 
do we want the WAL dir on tmpfs be optional and its size configurable ?